src/Command/ImportScorerCommand.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\Command;
+
+use App\Command\Adapter\ProviderInterface;
+use App\Entity\Scorer;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+class ImportScorerCommand extends Command
+{
+    CONST OVERWRITE = '--overwrite';
+    CONST FORCE = '--force';
+
+    protected $encoder;
+    protected $objectManager;
+    protected $stopwatch;
+    protected $input;
+    protected $output;
+    protected $dataProvider;
+    protected $formatFactory;
+    protected $path;
+    protected $symfonyStyle;
+
+    public function __construct(UserPasswordEncoderInterface $encoder, ObjectManager $objectManager,
+                                Stopwatch $stopwatch, ProviderInterface $provider)
+    {
+        $this->encoder = $encoder;
+        $this->objectManager = $objectManager;
+        $this->stopwatch = $stopwatch;
+        $this->dataProvider = $provider;
+        parent::__construct('app:import-scorer');
+    }
+
+    /**
+     *  configure options and argument of the command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Import scorers data from a CSV file')
+            ->setHelp('CSV file HEADERS: \'username,password\'');
+        $this->addOption(
+            'force',
+            'f',
+            InputOption::VALUE_NONE,
+            'Causes data ingestion to be applied into storage'
+        );
+        $this->addOption(
+          'overwrite',
+            'o',
+            InputOption::VALUE_NONE,
+            'overwrite Scorer in database'
```

Comment:
I think it's not required to have both parameters. The goal is to run the command in `dry-run` mode which means that it get executed on a regular way without saving anything into the database and have the `--force` attribute which will the persisting.
It doesn't make sense to run this command without the `--overwrite` attribute as nothing will happen.